transports/jms/src/test/java/org/mule/compatibility/transport/jms/integration/JmsRequestResponseTestCase.java

```diff
@@ -25,7 +25,7 @@ protected String getConfigFile() {
   @Test
   public void testNotSendiningCorrelationIDWithTemporaryQueue() throws Exception {
     InternalMessage response =
-        flowRunner("JMSNoCorrelationIDTemporaryQueue").nonBlocking().withPayload(TEST_MESSAGE).run().getMessage();
+        flowRunner("JMSNoCorrelationIDTemporaryQueue").withPayload(TEST_MESSAGE).run().getMessage();
```

Comment:
It was decided, at least for now, not to support non-blocking in transports in 4.0. So both support/tests have been removed.